en/docs/api-platform-gateway/build-gateway-with-custom-policies.md

```
@@ -0,0 +1,170 @@
+
+    ```powershell
+    ap --version
+    ```
+
+## Configure the Build File
+
+The `build.yaml` file is already included in the unzipped gateway package. It declares the gateway version and the list of policies to include. Edit this file to add your custom policies before building the gateway image.
+
+### Structure
+
+```yaml
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)